### PR TITLE
Travis CI Integration

### PR DESCRIPTION
**Pull request by mirons-google**
A first pass at an implementation of Travis integration.
- This adds a helper script to download protoc and install it correctly.
- This adds a badge to the main README

#14

### CODE REVIEW

File: scripts/travis_install.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# limitations under the License.
+
+# A script to perform the download and installation of protobuf.
+
+set -e
+
+echo "Downloading protoc."
+mkdir -p $PROTOC_DOWNLOAD_PATH
+cd $PROTOC_DOWNLOAD_PATH
+wget https://github.com/google/protobuf/releases/download/v3.2.0/protoc-3.2.0-linux-x86_64.zip
```
Comment (blueandgold) [inline scripts/travis_install.sh]: nit: Please feel free to pass on this, or do it later.  My nit is that we have 2 places where we are specifying which protoc to download.  So, we might update one, but forget to update here.  So, is there anyway that we can specify the protoc once, and both this and forseti-instance.py would be updated?  Again, not critical or blocking.

https://github.com/GoogleCloudPlatform/forseti-security/blob/91621f3109d3339b1da9a1eb08e7fdbe36ecc692/deployment-templates/py/forseti-instance.py#L121

Comment (blueandgold) [inline scripts/travis_install.sh]: At the least, can we put a comment near L121 in forseti-instance.py to say, "if you change the protoc version, please also update in travis_install.sh"?

Comment (mirons-google) [inline scripts/travis_install.sh]: Good point. One way might be to check-in a file to the repo containing the version/URL that these two scripts fetch.

Comment (carise) [inline scripts/travis_install.sh]: Yeah, that would be a good idea -- a cfg file that lists the required third party binaries/urls

Comment (mirons-google) [inline scripts/travis_install.sh]: I added #115

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**